doc/command_line.rst

```diff
@@ -16,13 +18,10 @@ Any task can be instantiated and run from the command line:
         def run(self):
             print self.x + self.y
 
-You can run this task from the command line like this::
+Should be run like this
 
-    $ python my_task.py MyTask --local-scheduler --x 123 --y 456
+    $ luigi --module my_module MyTask --x 123 --y 456 --local-scheduler
 
-The other way to run a Luigi task is to use the builtin *luigi* script.
-This will be default on your path and
-can be run by providing a module name.
-The module will imported dynamically::
+Or alternatively like this:
 
-    $ luigi --module my_module MyTask --x 123 --y 456 --local-scheduler
+    $ python -m luigi --module my_module MyTask --x 100 --local-scheduler
```

Comment:
Nope. This works in fact, see https://github.com/spotify/luigi/pull/1194. I ran something like `PYTHONPATH='' pip install --upgrade $HOME/spotify/repos/luigis/github_luigi/` when I tested this. I think you can also specify a git url pointing direclty to github's HEAD.

But it's expected to not work with latest stable version (we should release 2.0.0 soon)